packages/aws-cdk/lib/commands/diff.ts

```diff
@@ -35,10 +54,11 @@ export function printStackDiff(
   stackName?: string,
   changeSet?: DescribeChangeSetOutput,
   isImport?: boolean,
-  stream: FormatStream = process.stderr,
-  nestedStackTemplates?: { [nestedStackLogicalId: string]: NestedStackTemplates }): number {
+  nestedStackTemplates?: { [nestedStackLogicalId: string]: NestedStackTemplates }): { stackDiffCount: number, printableStackDiff: string } {
```

Comment:
jsdoc needs updated.

`stackDiffCount` reads weird to me `numStacksWithChanges` would be clearer
`printableStackDiff` -> `formattedDiff` 

also "stack" implies a single stack but this seems to be multiple